src/main/kotlin/lotto/domain/JackpotLevel.kt

```diff
@@ -0,0 +1,17 @@
+package lotto.domain
+
+enum class JackpotLevel(val matchCount: Int, val price: Int) {
+    NO_MATCH(0, 0),
+    ONE_MATCH(1, 0),
+    TWO_MATCH(2, 0),
+    THREE_MATCH(3, 5000),
+    FOUR_MATCH(4, 50000),
+    FIVE_MATCH(5, 1500000),
+    SIX_MATCH(6, 2000000000);
```

Comment:
```suggestion
    THREE_MATCH(3, 5_000),
    FOUR_MATCH(4, 50_000),
    FIVE_MATCH(5, 1_500_000),
    SIX_MATCH(6, 2_000_000_000);
```
언더스코어를 활용하여 가독성을 높일수 있어요 😄

Reply:
헉 이런 방법이 있었군요 감사합니다!! ㅎㅎ😊😊😊